ietf/templates/nomcom/nomcom_public_base.html

```diff
@@ -9,13 +9,16 @@
 {% endblock %}
 {% block content %}
     {% origin %}
+    {% with parts=request.path|split:'/' %}
+    {% with l0=parts|slice:'1:'|last l1=parts|slice:'-2:-1'|last %}
+    {% with selected=l0|default:l1 %}
```

Comment:
Clever but pushing the limits of what's a good idea in template logic. Might be better as a template filter if there's any chance we'd want this logic elsewhere.